model/metrics/process-metrics.yaml

```diff
@@ -100,3 +101,11 @@ groups:
     unit: "{fault}"
     attributes:
       - ref: process.paging.fault_type
+
+  - id: metric.process.uptime
+    type: metric
+    metric_name: process.uptime
+    stability: experimental
+    brief: "The number of seconds the process has been running."
```

Comment:
I agree "time" is better here.

Precision isn't currently part of the metrics spec, so we can't really specify it now. It also might not be good to specify. Different systems might capture this with different precision, so I don't think we'd want to force a value here.

Reply:
I understand that it could be hard to measure things with actual millisecond precision, but getting current time from the platform in milliseconds is possible on all system. So why not recommend millisecond precision here? What's the downside?

Reply:
I'll add in the suggested note, I agree that will be good